jetbrains-rider/src/software/aws/toolkits/jetbrains/ui/wizard/RiderSamProjectGenerator.kt

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.ui.wizard
+
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.ui.ColorUtil
+import com.intellij.ui.components.JBScrollPane
+import com.intellij.ui.components.JBTabbedPane
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.UIUtil
+import com.intellij.xml.util.XmlUtil
+import com.jetbrains.rdclient.util.idea.toVirtualFile
+import com.jetbrains.rider.ideaInterop.fileTypes.sln.SolutionFileType
+import com.jetbrains.rider.projectView.actions.projectTemplating.backend.ReSharperTemplateGeneratorBase
+import com.jetbrains.rider.projectView.actions.projectTemplating.impl.ProjectTemplateTransferableModel
+import com.jetbrains.rider.projectView.nodes.ProjectModelNode
+import com.jetbrains.rider.ui.themes.RiderTheme
+import com.jetbrains.rider.util.idea.application
+import software.aws.toolkits.resources.message
+import java.awt.Dimension
+import java.io.File
+import javax.swing.JScrollPane
+import javax.swing.JTabbedPane
+import javax.swing.JTextPane
+
+class RiderSamProjectGenerator(
+    private val samGenerator: SamProjectGenerator,
+    item: ProjectModelNode?,
+    group: String,
+    categoryName: String,
+    model: ProjectTemplateTransferableModel
+) : ReSharperTemplateGeneratorBase(
+    model = model,
+    createSolution = true,
+    createProject = true,
+    item = item) {
+
+    private val samPanel: SamInitSelectionPanel
+
+    private val tabb: JTabbedPane
```

Comment:
Call it projectStructure or something along those lines?

Reply:
Renamed to `projectStructurePanel`.